static/components/AroundCards.js

```diff
@@ -37,7 +37,7 @@ class App extends React.Component {
           userPosition = [position.coords.longitude, position.coords.latitude]
           focusUser(userPosition)
           this.setState({
-            userPosition: [userPosition[1], userPosition[0]]
+            userPosition: [userPosition[0], userPosition[1]]
```

Comment:
is this latitude, longitude?
I would suggest to [go for standards](http://stackoverflow.com/questions/7309121/preferred-order-of-writing-latitude-longitude-tuples?answertab=votes#tab-top), even though some of the tools we use have flipped them.